ddsp/training/train_util.py

```diff
@@ -230,6 +237,9 @@ def train(data_provider,
           tf.summary.scalar('losses/{}'.format(k), metric.result(), step=step)
           metric.reset_states()
 
+      # Report metrics for hyperparameter tuning.
```

Comment:
I would add a kwarg to `train()` called 'report_loss_to_hypertune=False` and then wrap this line in an if statement.

Then you can set that kwarg based on the flag in ddsp_run.py